M2/Macaulay2/m2/reals.m2

```diff
@@ -427,14 +428,19 @@ isReal CC := z -> imaginaryPart z == 0
 
 acosh = method()
 acosh Number := z -> log(z+sqrt(z^2-1))
+acosh Constant := acosh @@ numeric
 asinh = method()
 asinh Number := z -> log(z+sqrt(z^2+1))
+asinh Constant := asinh @@ numeric
 atanh = method()
 atanh Number := z -> log((1+z)/(1-z))/2
+atanh Constant := atanh @@ numeric
 acoth = method()
 acoth Number := z -> atanh(1/z)
+acoth Constant := acoth @@ numeric
```

Comment:
Are these two not exported?

Reply:
looks like it. by two you mean acoth and?

Reply:
atanh, right above it.